fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/api/entity/event/v1/ServerPlayerEvents.java

```diff
@@ -45,6 +47,31 @@
 		}
 	});
 
+	/**
+	 * An event that is called when a player takes fatal damage.
+	 *
+	 * <p>Mods can cancel this to keep the player alive.
+	 *
+	 * <p>Vanilla checks for player health <= 0 each frame (with {@link LivingEntity#isDead()}), and kills if true -
```

Comment:
```suggestion
	 * <p>Vanilla checks for player health {@code <= 0} each tick (with {@link LivingEntity#isDead()}), and kills if true -
```
This shouldn't be frame. This is server player event, so it should be a tick. If it's a frame as in fps on client, move this to client events.

Reply:
Whoops, thanks.